lib/changePWLinks.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const changePWLinks = {
+  "MySpace": "https://myspace.com/settings/profile/password",
+  "LinkedIn": "https://www.linkedin.com/psettings/change-password",
+  "Dubsmash": "https://dubsmash.com/reset-password",
+  "Canva": "https://www.canva.com/account",
+  "MyFitnessPal": "https://www.myfitnesspal.com/account/change_password",
+  "Adobe": "https://account.adobe.com/security",
+  "Dropbox": "https://www.dropbox.com/account/security",
+  "Houzz": "https://www.houzz.com/changePassword",
+  "Evite": "https://www.evite.com/settings/password",
+  "CafePress": "https://members.cafepress.com/account/profile.aspx",
```

Comment:
These might be problematic to keep up to date since they could change endpoints at any time.

Reply:
Yup. This will be a first experiment of the feature. We'll measure how many visitors actually use the feature before we decide if it's worth keeping links up-to-date.